### DISCUSSION TITLE
Is "jax.config.omnistaging_enabled" no longer present in the latest version ?

### DISCUSSION THREAD

**Discussion by abhiroop513** [jax-ml/jax#12273] [General] [answered]
I find "jax.config.omnistaging_enabled" no longer present in the latest version.  What is the alternative for this ? What do I replace it with ?

**Comment by jakevdp** [comment, marked as answer]
Thanks for the question: `omnistaging_enabled` was removed in https://github.com/google/jax/commit/dbae3e5ed15bdacd5c535fc75550e54b33c75d7e (JAX v0.3.15). It had been deprecated for over a year, and has had no effect since JAX v0.2.12 (see https://github.com/google/jax/pull/6196).
